hack/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The Crossplane Authors.
+Copyright 2021 The Crossplane Authors.
```

Comment:
@negz I also moved this copyright matter to https://github.com/crossplane/provider-alibaba/pull/62. Thanks.